tests/framework/user_guide/ravenTutorial/RomLoad.xml

```diff
@@ -23,11 +23,7 @@
   </Files>
 
   <Models>
-    <ROM name="rom" subType="NDinvDistWeight">
-      <Features>sigma-A,decay-A</Features>
-      <Target>A, time</Target>
-      <p>3</p>
-    </ROM>
+    <ROM name="rom" subType="pickledROM" />
```

Comment:
This has been modified since the ROM should be a pickledROM. I suggest to error out in case we try to overwrite a ROM that has a subtype /= pickledROM

Reply:
I agree.